scripts/install_cmake.sh

```diff
@@ -6,16 +6,18 @@
 
 set -e
 
-VERSION=3.7.1
+VERSION_MAJOR=3
+VERSION_MINOR=10
```

Comment:
Maybe I was a bit unclear on this: I'd be fine with using the latest stable cmake version here in this script.
With only requiring 3.10 I only meant above in the documentation change.
But it's fine either way in any case.